src/components/Mandal/MandalHeader/ShareButton.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import InviteModal from "./InviteModal";
+
+const ShareContainer = styled.div`
+  position: relative;
+  width: 50px;
+  height: 50px;
+
+  img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+`;
+
+export default function ShareButton() {
+  const [showInvite, setShowInvite] = useState(false);
+
+  const handleClickButton = () => {
```

Comment:
```suggestion
  const handleButtonClick = () => {
```
다른 부분과 통일성있게 적어주시면 더 좋을 것 같습니다!

Reply:
제가...왜 이렇게 했더라....? 수정하겠습니다!!